lib/setup.py

```diff
@@ -85,7 +100,7 @@ def run(self):
     package_data={"streamlit": ["py.typed", "hello/**/*.py"]},
     packages=setuptools.find_packages(exclude=["tests", "tests.*"]),
     # Requirements
-    install_requires=requirements,
+    install_requires=INSTALL_REQUIRES,
```

Comment:
Should we also convert setup.py to setup.cfg?
https://towardsdatascience.com/setuptools-python-571e7d5500f2

Reply:
I do think it'd be nice to have these things defined in a config file as opposed to code, but I don't think we'll see any immediate gains from doing so now aside from things being slightly cleaner, so I'd push this off as not necessary to do until we get a more compelling reason